app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= @recipe.name %>
+<%= @recipe.complexity %>
+<%= @recipe.genre %>
+<% @recipe.ingredients.each do |i| %>
+    <%= i.name %>
+<% end %>
+<%= @recipe.total_cost %>
```

Comment:
This is great but your code is missing the required HTML. This is still an HTML file, so HTML is expected.

```erb
<h1>Name: <%= @recipe.name %></h1>
<h2>Complexity: <%= @recipe.complexity %></h2>
<h3>Genre: <%= @recipe.genre %></h3>

<h4>Ingredients</h4>

<ul id="recipe_ingredients">
  <% @recipe.ingredients.each do |i| %>
    <li><%= i.name %></li>
  <% end %>
</ul>

<p>Total Cost: <%= @recipe.total_cost %></p>
```

These are just some HTML suggestions, you can use any others that make sense.